tests/conftest.py

```diff
@@ -851,3 +888,41 @@ def mock_events(api_url):
         status=200,
         body=response_body
     )
+
+
+@pytest.fixture
+def mock_upgrade(api_url, account_id, app_id):
```

Comment:
Another nitpick: could you rename this to something like `mock_account_management` since it covers both upgrading and downgrading?